<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_details</class>
 <widget class="QWidget" name="C_details">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>919</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>919</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>919</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Détail du film</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);

</string>
  </property>
  <widget class="QLabel" name="lbl_front">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>311</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_titre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_titreOriginal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titre original :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_dateReal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de réalisation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_genre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_adulte">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>510</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Catégorie adulte  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_resume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Résumé : </string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_resum">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>601</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enregistrer">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>ENREGISTRER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifier">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>MODIFER</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_stockage">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>550</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_stockge">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>530</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lieu de stockage:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>470</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_dateEnregistrement">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>450</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date d'enregistrement : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_suppression">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;
</string>
   </property>
   <property name="text">
    <string>SUPPRIMER</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_language">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>460</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Langue :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_id_distant">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>490</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant mondial :</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>760</y>
     <width>601</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="title">
    <string>Informations complémentaires</string>
   </property>
   <widget class="QLabel" name="lbl_acteur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Acteur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_metteurEnScene">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Met. en scène :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_acteur">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>421</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 255);
border-width: 1px;
border-style: solid;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_realisateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Réalisateur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_producteur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Producteur :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_acteur">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_metteurEnScene">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_realisateur">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_producteur">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_metteurEnScene">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_realisateur">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_producteur">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txt_titre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_titreOri">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_real">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_genre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_langue">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>460</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_id_distant">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>490</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_id_local">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>520</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant local :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_id_local">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>520</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_dispo">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_T_note">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>229</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>Note:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_note">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_T_vote">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>Nb. votes :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vote">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_stock">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_stock">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>550</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lieu de stockage :</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_adult">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>510</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
